Update crawl-site-content.md (#1093)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/crawl-site-content.md

# Manually request crawling and re-indexing of a site, a library or a list

> In SharePoint Online, content is automatically crawled based on a defined crawl schedule. The crawler picks up content that has changed since the last crawl and updates the index. You will want to manually request crawling and full re-indexing of a site, a document library, or a list after a schema change has occurred. 

Reviewer: ```suggestion
In SharePoint Online, content is automatically crawled based on a defined crawl schedule. The crawler picks up content that has changed since the last crawl and updates the index. You need to manually request crawling and full re-indexing of a site, a document library, or a list after a schema change has occurred. 
```

> In SharePoint Online, content is automatically crawled based on a defined crawl schedule. The crawler picks up content that has changed since the last crawl and updates the index. You will want to manually request crawling and full re-indexing of a site, a document library, or a list after a schema change has occurred. 

Reviewer 2: ```suggestion
In SharePoint Online, the content index is automatically updated, based on a defined crawl schedule that picks up any new changes. Even so, you should manually request crawling and full re-indexing after you make any schema changes to a site, a document library, or a list .

> [!CAUTION]
>  Re-indexing a site can cause a massive load on the search system. Don't re-index your site unless you've made changes that require all items to be re-indexed. 

## Re-index after changing managed properties
<a name="__top"> </a>

When people search for content on your SharePoint sites, what's in your search index decides what they'll find. The search index contains information from all documents and pages on your site. 
  
The search index is built up by crawling the content on your SharePoint site. The crawler picks up content and metadata from the documents in the form of crawled properties. To get the content and metadata from the documents into the search index, the crawled properties must be mapped to managed properties. Only managed properties are kept in the index. This means that users can only search on managed properties.
  

> When you have changed a managed property, or when you have changed the mapping of crawled and managed properties, the site must be re-crawled before your changes will be reflected in the search index. Because your changes are made in the search schema, and not to the actual site, the crawler will not automatically re-index the site. To make sure that your changes are crawled and fully re-indexed, you must request a re-indexing of the site. The site content will be re-crawled and re-indexed so that you can start using the managed properties in queries, query rules and display templates.

Reviewer 2: ```suggestion
When you have changed a managed property, or when you have changed the mapping of crawled and managed properties, the site must be re-crawled before your changes will be reflected in the search index. Because your changes are made in the search schema, and not to the actual site, the crawler will not automatically re-index the site. To make sure that your changes are crawled and fully re-indexed, you must manually request a re-indexing of the site. After the site content has been re-crawled, you can start using the managed properties in queries, query rules and display templates.
```

> You can also choose to only re-index a document library or a list. When you have changed a managed property that's used in a library or list, or changed the mapping of crawled and managed properties, you can specifically request a re-indexing of that library or list only. All of the content in that library or list is marked as changed, and the content is picked up during the next scheduled crawl and re-indexed.

Reviewer 2: ```suggestion
You can also choose to only re-index a specific document library or a list. When you change a managed property that's used in a library or list, or if you change the mapping of crawled and managed properties, you can specifically request a re-indexing of only that library or list. All of the content in that library or list will then be marked as changed, picked up during the next scheduled crawl, and re-indexed.
```
